### DISCUSSION TITLE
Keyball61 右側の上から３段目、横一列のキーが反応しなくなってしまいました

### DISCUSSION THREAD

**Discussion by escape010** [Yowkees/keyball#624] [Q&A]
お世話になります、
半年ほど問題なく使用していましたが、突然keyball61 右側の上から３段目、横一列のキーが反応しなくなってしまいました。
どの様に対応して良いか分かりかねてしまうため、お手数ですが対応方法を教えていただけますと幸いです。
尚、当方電子工作はこのKeyballが初めての初心者です。

お手数ですが、よろしくお願い致します。
![IMG_1734](https://github.com/user-attachments/assets/b4218308-6d46-424e-afa3-90800bee6946)
![IMG_1735](https://github.com/user-attachments/assets/8268fddc-ce50-40ab-bb95-6ae043f12770)
<img width="1167" alt="Screenshot 2024-07-14 at 13 27 58" src="https://github.com/user-attachments/assets/e7dba8f8-5618-453b-879e-19819baf08a4">

**Comment by Ryo624** [comment]
縦、横一列でキーが効かない場合はProMicroのはんだ付けを疑います。
今回の反応のないキー一列はROW２という端子とつながっていますので、こちらのはんだ付けを見直してみてください。
上から見た状態なので確実とは言えませんが、はんだが浮いて端子に上手く馴染んでいないように見えるので山型に端子に馴染ませるように意識してみてください。
